app-sso/service-operators/tls-scenario-guides.hbs.md

```diff
@@ -0,0 +1,785 @@
+  issuerRef:
+    name: letsencrypt-production
+    kind: ClusterIssuer
+  secretName: sso-cert
+
+---
+apiVersion: sso.apps.tanzu.vmware.com/v1alpha1
+kind: AuthServer
+metadata:
+  name: sso
+  namespace: login
+  annotations:
+    sso.apps.tanzu.vmware.com/allow-unsafe-identity-provider: ""
+    sso.apps.tanzu.vmware.com/allow-client-namespaces: "*"
+spec:
+  #! --- TLS ---
+  tls:
+    certificateRef:
+      name: sso
+  #! -----------
+  identityProviders:
+    - name: test-users
+      internalUnsafe:
+        users:
+          - username: user
+            password: password
+  tokenSignature:
+    signAndVerifyKeyRef:
+      name: signing-key
+
+---
+apiVersion: secretgen.k14s.io/v1alpha1
+kind: RSAKey
+metadata:
+  name: signing-key
+  namespace: login
+spec:
+  secretTemplate:
+    type: Opaque
+    stringData:
+      key.pem: $(privateKey)
+      pub.pem: $(publicKey)
+```
+
+### … in another `Namespace`
+
+Thanks to [secretgen-controller](https://github.com/vmware-tanzu/carvel-secretgen-controller) we can export and
+import `Secrets` across namespaces. When our `Certificate` is located in another namespace - because maybe it's
+controlled by another team - then its `Secret` can be offered to other namespaces for import.
+
+Pay attention to the use of `SecretExport` and `SecretImport` to facilitate the transfer across namespaces.
+
+```yaml
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: tls
+
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: sso
+  namespace: tls
+spec:
+  dnsNames:
+    - "sso.login.my-tap.example.com"
+  issuerRef:
+    name: letsencrypt-production
+    kind: ClusterIssuer
+  secretName: sso-cert
+
+---
+apiVersion: secretgen.carvel.dev/v1alpha1
+kind: SecretExport
+metadata:
+  name: sso-cert
+  namespace: tls
+spec:
+  toNamespace: login
+
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: login
+
+---
+apiVersion: secretgen.carvel.dev/v1alpha1
+kind: SecretImport
+metadata:
+  name: sso-cert
+  namespace: login
+spec:
+  fromNamespace: tls
+
+---
+apiVersion: sso.apps.tanzu.vmware.com/v1alpha1
+kind: AuthServer
+metadata:
+  name: sso
+  namespace: login
+  annotations:
+    sso.apps.tanzu.vmware.com/allow-unsafe-identity-provider: ""
+    sso.apps.tanzu.vmware.com/allow-client-namespaces: "*"
+spec:
+  #! --- TLS ---
+  tls:
+    secretRef:
+      name: sso-cert
+  #! -----------
+  identityProviders:
+    - name: test-users
+      internalUnsafe:
+        users:
+          - username: user
+            password: password
+  tokenSignature:
+    signAndVerifyKeyRef:
+      name: signing-key
+
+---
+apiVersion: secretgen.k14s.io/v1alpha1
+kind: RSAKey
+metadata:
+  name: signing-key
+  namespace: login
+spec:
+  secretTemplate:
+    type: Opaque
+    stringData:
+      key.pem: $(privateKey)
+      pub.pem: $(publicKey)
+```
+
+## I have an existing TLS certificate
+
+If you have an existing TLS certificate and private key, then this scenario is yours.
```

Comment:
**[suggestion]** The tone is different from the other scenarios, make it more formal.
**[suggestion]** Add "For example, your certificate was created outside the cluster".
**[suggestion]** Refer to the "troubleshooting" section suggested in the above scenario.